### DISCUSSION TITLE
Unable to update load to connection through Python API

### DISCUSSION THREAD

**Discussion by s4lmonsushi** [idea-statica/ideastatica-public#615] [Help Q&A]
Hi,

I have created a standard web side plate connection and trying to update loads through python using the API.

Following the example "parameters-anchors.py", I was able to locate the correct loaded segment and add load into the segment correctly.

However, when run the script, the bolt results was the same for each load update loop, although the loads have been updated for each loop:
![image](https://github.com/user-attachments/assets/59d8b59c-13d6-43e4-9486-4d49b57d264c)

I am also having trouble with the "ideaConnectionClient.GetCheckResultsJSON", for both my script and the "T-stub.py" example script. For some reason it returns as an error:
![image](https://github.com/user-attachments/assets/cce5af6a-462c-42aa-9a94-75328768ff5a)

Does anyone know how to get around this issue?

**Comment by s4lmonsushi** [comment]
I have tried to update IdeaStatica to the latest version and see if this issue can be resolved. However after updating I am seeing this message:

![image](https://github.com/user-attachments/assets/f70667fe-1ba7-4899-94e2-7897799ead14)

**Reply by nathanlukeIDEA** [reply]
Hi @s4lmonsushi, we have an API issue in the latest version that we are fixing. Please revert to the previous version. Can you please send me your files and I will test it.
